src/components/chat/chat.tsx

```diff
@@ -38,6 +44,27 @@ export const Chat: FC<Props> = ({ roomId, room }) => {
     setMessages((prev) => [...prev, newMessage]);
   };
 
+  const handleMessage = (event: { target: { value: string } }) => {
+    const inputValue = event.target.value;
+
+    validateForm(inputValue);
+
+    if (!messageError && inputValue.length <= 150) {
+      setMessage(inputValue);
+    } else if (inputValue.length > 150) {
+      setMessage(inputValue.slice(0, 150));
+    }
+  };
+
+  const [messageError, setValidation] = useState("");
+
+  const validateForm = (newMessage: string) => {
+    const error = !validMessage.safeParse(newMessage).success
+      ? textLength.chatMessage.error
+      : "";
```

Comment:
```suggestion
    const error = validMessage.safeParse(newMessage).success
      ? ""
      : textLength.chatMessage.error;
```

Reply:
I feel like changes like this are unnecessary, hard to argue one is clearer than the other